packages/storybook/.storybook/config.js

```diff
@@ -1,7 +1,7 @@
 import { configure } from '@kadira/storybook';
 
 function loadStories() {
-  require('../stories/index.js');
```

Comment:
FYI I turned this requirement off: https://github.com/orion-ui/orion/blob/67de5837d47eafafd6a02578d8dfa3d2c0be0075/packages/eslint-config-orion/index.js#L11